app/presenters/courses_presenter.rb

```diff
@@ -59,6 +59,10 @@ def can_remove_course?
     @can_remove ||= current_user&.admin? || campaign_organizer?
   end
 
+  def can_delete_course?
```

Comment:
As this will behave the same way as `can_remove?`, I suggest using `alias_method` along with a comment, rather than repeating the implementation.